.env

```diff
@@ -18,6 +18,8 @@ HASH_PG_PASSWORD=postgres
 HASH_PG_PORT=5432
 HASH_PG_USER=postgres
 
+HASH_MIGRATION_PG_URL=postgres://postgres:postgres@localhost/postgres
```

Comment:
Should we consider moving to a PG URI for the `HASH_PG` env vars above? Obviously a follow-up task, but I think that would also make deployment a little easier as we don't have to manage 5 env vars that can be handled in one

Reply:
Personally I prefer declaring PG env vars individually (host, username, password, etc). The only reason I was forced to use a URL here is because the `node-pg-migrate` client only supports reading the full URL as an environment variable, not the individual pieces that make up that URL.

But whether we supply the variables individually, or as part of a single URL I think we should use one or the other - not require both (5 env vars) to be set. I think this should be addressed as part of the task I mentioned [in my other comment](https://github.com/hashintel/hash/pull/438#discussion_r838953653).